File: Algorithms /fuzzbuzz.java
public class fuzzbuzz {
	public static void fuzzMethod(int num){
		if(num % 5 == 0){
			if(num % 3 == 0){
				System.out.println("Fuzzbuzz");
			}
		} else if(num % 3 == 0){
			System.out.println("Fizz");
		} else if(num % 5 == 0){
			System.out.println("Buzz");
		}
	}
	public static void main(String[] args){
		int num = 30;
		System.out.println(fuzzMethod(30));
	}
}

File: Companies/Everlane/fuzzbuzz.js
var fuzzbuzz = function(num){
	for(var i = 0; i < num;i++){
		if(i % 5 === 0){
			if(num % 3 === 0){
				console.log("Fuzzbuzz");
			}
		} else if(i % 5 === 0){
			console.log("Buzz");
		} else if(i % 3 === 0){
			console.log("Fuzz");
		} else {
			console.log(i);
		}
	}
};

fuzzbuzz(20);